socorro/collector/wsgi_generic_collector.py

```diff
@@ -64,10 +64,18 @@ def _process_fieldstorage(self, fs):
             return fs
 
     #--------------------------------------------------------------------------
+    def _is_content_gzipped(self):
+        return web.ctx.env.get('HTTP_CONTENT_ENCODING') == 'gzip'
+
+    #--------------------------------------------------------------------------
+    def _get_content_length(self):
+        return web.ctx.env.get('CONTENT_LENGTH', 0)
```

Comment:
I added these here because in the tests we mock the bejesus out of `web` in this module and it simplified everything if we kept web usage here.